### PR TITLE
Allow emoji on popup label

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by silverwind** [comment]
Does it actually work? I thought you need to call `emojify.run(element)` on dynamically inserted DOM nodes.

File: web_src/js/features/contextPopup.js [hunk 1]
```diff
@@ -20,14 +20,15 @@ function issuePopup(suburl, owner, repo, index, $element) {
...
+      const labelName = emojify.replace(label.name);
       const red = parseInt(label.color.substring(0, 2), 16);
       const green = parseInt(label.color.substring(2, 4), 16);
       const blue = parseInt(label.color.substring(4, 6), 16);
       let color = '#ffffff';
       if ((red * 0.299 + green * 0.587 + blue * 0.114) > 125) {
         color = '#000000';
       }
-      labels += `<div class="ui label" style="color: ${color}; background-color:#${label.color};">${label.name}</div>`;
+      labels += `<div class="ui label has-emoji" style="color: ${color}; background-color:#${label.color};">${labelName}</div>`;
```
Comment (silverwind) [inline web_src/js/features/contextPopup.js]:
```suggestion
      labels += `<div class="ui label" style="color: ${color}; background-color:#${label.color};">${labelName}</div>`;
```

No longer needed, it only works on page load.

Comment (jaqra) [reply web_src/js/features/contextPopup.js]: I thought that specify text inside. Done as you wish :)

**Review by silverwind — APPROVED** [review]

**Review by guillep2k — APPROVED** [review]

File: web_src/js/features/contextPopup.js (hunk 1 above)
Comment (silverwind) [reply web_src/js/features/contextPopup.js]: A more appropriate name would be `replace-emoji-tokens` or something.

**Pull request merged**